cli/src/commands/subgraph/commands/publish.ts

```diff
@@ -42,6 +48,10 @@ export default (opts: BaseCommandOptions) => {
     '--subscription-protocol <protocol>',
     'The protocol to use when subscribing to the subgraph. The supported protocols are ws, sse, and sse_post.',
   );
+  command.option(
+    '--websocket-subprotocol <protocol>',
+    'The subprotocol to use when subscribing to the subgraph. The supported protocols are auto, graphql-ws, and graphql-transport-ws.',
```

Comment:
4th?

Reply:
we are letting the user set the subprotocol in different commands like while creating, publishing and updating a subgraph or monograph so the repetition

Reply:
Done